fvm/src/kernel/default.rs

```diff
@@ -748,79 +745,51 @@ where
     }
 }
 
-fn draw_randomness(
-    rbase: &[u8; RANDOMNESS_LENGTH],
-    pers: i64,
-    round: ChainEpoch,
-    entropy: &[u8],
-) -> anyhow::Result<[u8; RANDOMNESS_LENGTH]> {
-    let mut state = Params::new().hash_length(32).to_state();
-    state.write_i64::<byteorder::BigEndian>(pers)?;
-    state.write_all(rbase)?;
-    state.write_i64::<byteorder::BigEndian>(round)?;
-    state.write_all(entropy)?;
-    state
-        .finalize()
-        .as_bytes()
-        .try_into()
-        .map_err(anyhow::Error::from)
-}
-
 impl<C> RandomnessOps for DefaultKernel<C>
 where
     C: CallManager,
 {
     fn get_randomness_from_tickets(
         &self,
-        personalization: i64,
         rand_epoch: ChainEpoch,
-        entropy: &[u8],
     ) -> Result<[u8; RANDOMNESS_LENGTH]> {
         let t = self.call_manager.charge_gas(
             self.call_manager
                 .price_list()
-                .on_get_randomness(entropy.len()),
+                // TODO
+                .on_get_randomness(),
         )?;
 
         // TODO(M2): Check error code
         // Specifically, lookback length?
```

Comment:
Let's handle this TODO and remove it. At this point, all we have to do is verify that the epoch is before the current epoch.